docs/coordinates/apply_space_motion.rst

```diff
@@ -45,12 +45,12 @@ Or, we can specify the new time to evaluate the position at::
 If the |SkyCoord| object has no specified radial velocity (RV), the RV is
 assumed to be 0. The new position of the source is determined assuming the
 source moves in a straight line with constant velocity in an inertial frame.
-There are no plans to support more complex evolution (e.g. non-inertial
-frames or more complex evolution), as that is out of scope for the Astropy core
-(although it may well be in-scope for a variety of affiliated packages).
+There are no plans to support more complex evolution (e.g., non-inertial
+frames or more complex evolution), as that is out of scope for the `astropy`
+core (although it may well be in-scope for a variety of affiliated packages).
 
-Example: Use velocity to compute sky position at different epoch
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Example: Use Velocity to Compute Sky Position at Different Epochs
```

Comment:
I'm not sure if I like having all words capitalized in titles - is this normally recommended for readability?

Reply:
I also went back and forth about this -- Chicago Manual of Style, which is generally the resource for academic and formal writing, recommends up style capitalization in headings, so that's what I went with, but I agree it can look strange in longer headings and sub-headings.

For the tutorials, since they are a little less formal, I've been doing up style capitalization in the first few sections of more structured "set up" headings, but then switching to down style in the "learning sections" which tend to have much longer headings.

Maybe we should do something similar in the documentation, as in up style for all of the more formal sections, but then down style in any longer "example" headings like the one you've pointed out here?

What do other people think about capitalization in headings?

Reply:
I'm personally a fan of capitalization. Maybe I've been brainwashed by the Chicago Manual (that's what I learned from in grade school...), but I've always had in my head that anything in a "heading" style that's not capitalized doesn't look right.

Reply:
So it sounds like maybe a compromise is best -- up style capitalization in formal headings and down style in longer "examples" type headings? I'll try that moving forward and see how everyone likes it!